azure-mgmt-resources/src/test/java/com/azure/management/resources/fluentcore/dag/ProxyTaskGroupTests.java

```diff
@@ -1451,10 +1452,14 @@ public void beforeGroupInvoke() {
          */
 
         final ArrayList<String> seen = new ArrayList<>();
+        CountDownLatch down = new CountDownLatch(1);
         itiC.getTaskGroup()
                 .invokeAsync(itiC.getTaskGroup().newInvocationContext())
                 .subscribe(indexable ->
-                        seen.add(indexable.getKey()));
+                                seen.add(indexable.getKey()),
+                        throwable -> down.countDown(),
+                        () -> down.countDown());
+        down.await();
```

Comment:
Should we use await with timeout?

Reply:
Without network connection, just keep the case simple.